### ISSUE TITLE
Increase filter to 10GB

### ISSUE THREAD

**Issue by stdavis** [agrc/broadband#85]
From Bert:
Something to stay on top of...we should modify their data and the map click results...I think we keep the filter the same for now

-------
Bert Granberg
Utah AGRC
@BertAGRC
http://gis.utah.gov

Begin forwarded message:

From: Kelleigh Cole <kcole@utah.gov>
Date: January 11, 2018 at 11:18:59 AM MST
To: Roger Timmerman <RTimmerman@utopiafiber.com>
Cc: "Granberg, Bert" <bgranberg@utah.gov>
Subject: Re: Change in UTOPIA Offered Speeds

Thanks for the update. That's crazy fast. The current platform taps out at 1 Gig so we'll talk about how to do this and get back to you.

On Thu, Jan 11, 2018 at 11:12 AM, Roger Timmerman <RTimmerman@utopiafiber.com> wrote:
Kelleigh,

Yesterday, UTOPIA Fiber launched a 10 Gbps residential offering, available anywhere in our coverage area.  Businesses can get up to 100 Gbps anywhere that UTOPIA Fiber is available.  Can we get our data updated to reflect this?  Here is some press related to the announcement, with more likely to come…

https://www.deseretnews.com/article/900007289/utopia-to-offer-fastest-internet-service-in-the-country.html

Thanks,

**Comment by stdavis** [comment]
@ZachBeck Do you know if our data reflect these higher speeds?

**Comment by stdavis** [comment]
@ZachBeck Ping, when you have a minute...

**Comment by ZachBeck** [comment]
Is this for the fiber hexagons?

**Comment by stdavis** [comment]
No, it's for `UBBMAP.UBBADMIN.BB_Service`, the data that feeds the residential BB map.

**Comment by ZachBeck** [comment]
BB_Service shows 10gb for utopia, last edited on 10/28/2019

**Comment by stdavis** [comment]
Here's what the sliders look like in production currently:
![image](https://user-images.githubusercontent.com/1326248/76448229-a3ee5480-638f-11ea-9f97-1c4a7f4cf016.png)

Here's the distribution of values in the data:
![image](https://user-images.githubusercontent.com/1326248/76448260-b10b4380-638f-11ea-8e38-406c014a157c.png)
![image](https://user-images.githubusercontent.com/1326248/76448280-b9fc1500-638f-11ea-9d7f-2b212e36445e.png)

Here's what the current state of the rewrite is:
![image](https://user-images.githubusercontent.com/1326248/76448295-c2545000-638f-11ea-835e-4c27cff73c6c.png)

We obviously can't have a slider stop for each unique value in the data. And I'm not sure if we want to hide some of the data by default. Perhaps the sliders should be something like this?

![image](https://user-images.githubusercontent.com/1326248/76449324-7c988700-6391-11ea-97b3-9784904f2239.png)

That way, no data would be hidden by default. @nathankota Thoughts? I know that Bert and Kelleigh had major opinions about this back when the first version was built, but I'm not sure if anyone cares now.

**Comment by nathankota** [comment]
I don't think opinions are as strong anymore, and I also do not think to bit of lost granularity at the low end is significant. Perhaps others disagree. I do think it is important to include the additional factor of 10 at the top of the range. Personally, I think the new slider idea works well.
